raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -2548,6 +2549,10 @@ public OptionalLong highWatermark() {
         }
     }
 
+    public Optional<Node> idToNode(int id, String listener) {
```

Comment:
Similar to my other comments but how about `voterNode` for the name of the method?

Reply:
ack